defs/browser.json

```diff
@@ -3379,5 +3384,110 @@
     "!type": "fn(width?: number, height?: number) -> +HTMLImageElement",
     "!url": "https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/Image",
     "!doc": "Image Element constructor. Accepts two optional parameters: Image([unsigned long width, unsigned long height]). Returns an HTMLImageElement instance just as document.createElement('img') would."
+  },
+  "Geolocation": {
```

Comment:
These aren't actual global variables, so I'd prefer you put them in the `!define` block, so that you can reference them, bu they won't show up as completions.